libcontainer/rootfs_linux.go

```diff
@@ -152,14 +153,46 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		}
 		return nil
 	case "tmpfs":
+		copyUp := m.Extensions&configs.EXT_COPYUP == configs.EXT_COPYUP
+		tmpDir := ""
 		stat, err := os.Stat(dest)
 		if err != nil {
 			if err := os.MkdirAll(dest, 0755); err != nil {
 				return err
 			}
 		}
-		if err := mountPropagate(m, rootfs, mountLabel); err != nil {
-			return err
+		if copyUp {
+			tmpDirPrefix := os.Getenv("XDG_RUNTIME_DIR")
```

Comment:
Just use the container's /tmp dir or some hidden dir inside the rootfs to make this.  Since its temporary and is only there to make the initial tmpfs destination it can be anywhere or even back in your container's /run.